src/torchrunx/launcher.py

```diff
@@ -208,7 +218,8 @@ def run(
             hostnames=self.hostnames,
             worker_world_size=worker_world_size,
             worker_global_ranks=worker_global_ranks,
-            worker_log_files=worker_log_files,
+            worker_log_names=worker_log_names,
```

Comment:
Do we need to pass the `worker_log_names`? Or can we just keep a convention in the worker to name according to `torchrunx.hostname.{worker_rank}`?

Reply:
and can we just name this like f"torchrunx.{hostname}[{local_rank}]"

Reply:
I'm going to have to make it f"torchrunx.{hostname}.[{local_rank}]", such that the agent logger is a parent of the worker loggers (there needs to be a dot).

Reply:
Does the agent logger need to be a parent of the worker loggers?

Reply:
I suppose not - though otherwise nothing really goes on in the agent logs.

Reply:
Ok, I've switched it over to this convention, so the agent loggers are no longer parents to the worker loggers.